scripts/opensea/create-listing.ts

```diff
@@ -0,0 +1,100 @@
+import { Chain, OpenSeaSDK } from "opensea-js";
+import { EnvironmentType, getEnvConfigById } from "@bosonprotocol/common";
+import { program } from "commander";
+import { providers, Wallet } from "ethers";
+import {
+  Wallet as WalletV6,
+  JsonRpcProvider as JsonRpcProviderV6
+} from "ethers-v6";
+import { EthersAdapter } from "../packages/ethers-sdk/src";
+import { CoreSDK } from "../packages/core-sdk/src";
+import { API_BASE_TESTNET } from "opensea-js/lib/constants";
+
+program
+  .description("Create a Listing on Opensea.")
+  .argument(
+    "<SELLER_PRIVATE_KEY>",
+    "Private key of the Seller account (assistant role)."
+  )
+  .argument("<TOKEN_ID>", "tokenId of the token to be listed")
+  .requiredOption("-k, --apiKey <OPENSEA_API_KEY>", "Opensea API Key")
+  .option("-e, --env <ENV_NAME>", "Target environment", "testing")
+  .option("-c, --configId <CONFIG_ID>", "Config id", "testing-80002-0")
+  .parse(process.argv);
+
+async function main() {
+  const [sellerPrivateKey, tokenId] = program.args;
+
+  const opts = program.opts();
+  const OPENSEA_API_KEY = opts.openseaApiKey;
+  const envName = opts.env || "testing";
+  const configId = opts.configId || "testing-80002-0";
+  const defaultConfig = getEnvConfigById(envName as EnvironmentType, configId);
+  const chainId = defaultConfig.chainId;
+  const provider = new providers.JsonRpcProvider(defaultConfig.jsonRpcUrl);
+  const providerV6 = new JsonRpcProviderV6(defaultConfig.jsonRpcUrl);
+  const sellerWallet = new Wallet(sellerPrivateKey);
+  const sellerWalletV6: WalletV6 = new WalletV6(sellerPrivateKey, providerV6);
+  const coreSDK = CoreSDK.fromDefaultConfig({
+    web3Lib: new EthersAdapter(provider, sellerWallet),
+    envName,
+    configId
+  });
+  const WETH_ADDRESS = "0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa";
```

Comment:
which chain?

Reply:
Changed. Now reading the WETH address from the coreSDK config